statsmodels/regression/tests/test_lme.py

```diff
@@ -943,6 +943,39 @@ def test_handle_missing():
                 assert_equal(len(result1.fittedvalues), result1.nobs)
 
 
+def test_summary_col():
+    from statsmodels.iolib.summary2 import summary_col
+    ids = [1,1,1,1,1,2,2,2,2,3,3,3,3,3,3]
+    x = [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15]
+    # hard coded simulated y
+    # ids = np.asarray(ids)
+    # np.random.seed(123987)
+    # y = x + np.array([-1, 0, 1])[ids - 1] + 2 * np.random.randn(len(y))
```

Comment:
minor speedup, and I needed to fix the case because for most seeds the optimization didn't converge properly (or ended up at corner or non-invertible hessian.)